### PR TITLE
Creator digest settings

### PR DESCRIPTION
**Pull request by cdolm92**
# What

Creator can now manage their notification preferences to receive individual or a digest of their project's pledge activity.

# See 👀
creatordigest

# TODO ⏰

- [ ] Need feedback on the design as it may not be the most elegant implementation

### CODE REVIEW

**Comment by cdolm92** [comment]
@justinswart @Scollaco @kickstarter/native-squad g2g?

File: Library/ViewModels/SettingsViewModel.swift [hunk 1]
```diff
@@ -453,6 +489,7 @@ private enum Notification {
   case mobilePostLikes
   case mobileUpdates
   case postLikes
+  case creatorTips
```
Comment (Scollaco) [inline Library/ViewModels/SettingsViewModel.swift]: Could you move `case creatorTips` up so we keep the enumeration in alphabetical order?

File: Library/ViewModels/SettingsViewModel.swift [hunk 2]
```diff
@@ -462,6 +499,7 @@ private enum Notification {
     case .follower, .mobileFollower:                return "New followers"
     case .friendActivity, .mobileFriendActivity:    return "Friend backs a project"
     case .postLikes, .mobilePostLikes:              return "New likes"
+    case .creatorTips:                              return "Creator tips"
```
Comment (Scollaco) [inline Library/ViewModels/SettingsViewModel.swift]: and here too

**Comment by Scollaco** [comment]
It looks good to me. I just added a couple of comments about minor things. Very good job here, though!

**Comment by cdolm92** [comment]
@kickstarter/native-squad g2g?

**Pull request merged**